opencog/query/Distributed/DistSCM.cc

```diff
@@ -0,0 +1,214 @@
+/*
+ * DistSCM.cc
+ *
```

Comment:
This appears to have nothing at all to do with query, I will merge this pull request, but then I will immediately move this code to a new directory.